PostBoard.Api/Services/BirthdayService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using PostBoard.Api.Models;
+using PostBoard.Api.Services.Contracts;
+
+namespace PostBoard.Api.Services;
+
+public class BirthdayService : IBirthdayService
+{
+    private readonly PostBoardContext _dbContext;
+
+    public BirthdayService(PostBoardContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+ 
+    public async Task<IList<Birthday>> GetAllAsync()
+    {
+        return await _dbContext.Birthdays.ToListAsync();
+    }
+
+    public async Task<Birthday> GetByIdAsync(int id)
+    {
+        return await _dbContext.Birthdays.FirstOrDefaultAsync(p => p.Id == id);
```

Comment:
Here you have to check if result birthday is null.
If it's null, you must throw an exception with correct message (`not found` message).